sbncode/LArRecoProducer/RangePAllPID_module.cc

```diff
@@ -49,9 +49,12 @@ class sbn::RangePAllPID : public art::EDProducer {
   art::InputTag fTrackLabel;
 
 };
-const static std::vector<int> PIDs {13, 2212};
-const static std::vector<std::string> names {"muon", "proton"};
+const static std::vector<int> PIDs {13, 211, 2212};
+const static std::vector<std::string> names {"muon", "pion", "proton"};
 
+// P.A. Zylaet al.(Particle Data Group), Prog. Theor. Exp. Phys.2020, 083C01 (2020)
```

Comment:
I am curious why this is hardcoded vs using the `TDatabasePDG` that is used in the CAFMaker, although I doubt it will change the physics output in this case.

Reply:
Fixed!